doc/ChangeLog.md

```diff
@@ -45,9 +45,13 @@ separate project.  Going forward Infix' focus is entirely on NETCONF.
   are supported.
 - Include Buildroot `legal-info` in releases, i.e., licenses, sources
   with patches, as well as csv files for packages and toolchain
+- Issue #369: Remove limitation that the routing instance must be
+  named 'default'
 
 ### Fixes
-
+- Issue #391 Creating VLAN interface in the CLI with "edit interface vlan<N>"
```

Comment:
Minor, we try to avoid HTML `<` and `>` in Markdown, unless we want to trigger actual HTML.   I suggest enclosing the config line like this `edit interface vlanN` and explain: "does not set VLAN id to N."